fe1-web/parts/lao/socialMedia/SocialMediaNavigation.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { Ionicons } from '@expo/vector-icons';
+import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
+import STRINGS from 'res/strings';
+import { red, gray } from 'styles/colors';
+
+import SocialHome from './SocialHome';
+import SocialFollows from './SocialFollows';
+import SocialSearch from './SocialSearch';
+import SocialProfile from './SocialProfile';
+
+const Tab = createMaterialTopTabNavigator();
+
+const SocialMediaNavigation = () => (
+  <Tab.Navigator
+    screenOptions={({ route }) => ({
+      tabBarIcon: ({ color }) => {
+        let iconName;
+
+        switch (route.name) {
+          case STRINGS.social_media_navigation_tab_home:
+            iconName = 'home';
+            break;
+          case STRINGS.social_media_navigation_tab_search:
+            iconName = 'search';
+            break;
+          case STRINGS.social_media_navigation_tab_follows:
+            iconName = 'people';
+            break;
+          case STRINGS.social_media_navigation_tab_profile:
+            iconName = 'person';
+            break;
+          default:
+            console.log('wrong route.');
```

Comment:
Is console.log the most appropriate there or is it better to throw an error ? I'm not sure to be honest, but it may be a good idea to think about it.

Reply:
I will change it to console.error but I am nearly sure that we will not enter the default case.